docs/AQL/master03-syntax.adoc

```diff
@@ -219,12 +219,36 @@ The table below shows the supported AQL comparison operators, meaning and exampl
 |>=         |Greater than or equal to   |`o/data[at0001]/..../data[at0003]/items[at0004]/value/value >=140`
 |<          |Smaller than               |`o/data[at0001]/.../data[at0003]/items[at0004]/value/value <160`
 |\<=        |Smaller than or equal to   |`o/data[at0001]/.../data[at0003]/items[at0004]/value/value <=160`
-|!=         |not equals to              |`c/archetype_details/template_id/value != ''`
-|matches    |advanced matcher           |`o/data[at0002]/.../name/defining_code/code_string matches {'18919-1', '18961-3', '19000-9'}`
+|!=         |Not equal to               |`c/archetype_details/template_id/value != ''`
+|LIKE       |Simple pattern matching    |`o/data[at0001]/.../name/value LIKE 'left %'`
+|matches    |Advanced matcher           |`o/data[at0002]/.../name/defining_code/code_string matches {'18919-1', '18961-3', '19000-9'}`
 |===
 
+==== LIKE
 
-=== matches operator definition
+The `LIKE` binary operator is used to compare a value of type string (or dates and times) against a simple pattern.
+The left operand is an AQL identified path, while the right operand is string value representing the pattern to be matched.
+The `LIKE` expression returns `true` if the value matches the supplied pattern.
+
+Below is an example using a simple pattern matching:
+--------
+SELECT
+   e/ehr_id/value, c/context/start_time
+FROM
+   EHR e
+      CONTAINS COMPOSITION c[openEHR-EHR-COMPOSITION.administrative_encounter.v1]
+         CONTAINS ADMIN_ENTRY admission[openEHR-EHR-ADMIN_ENTRY.admission.v1]
+WHERE
+   c/context/start_time LIKE '2019-%'
+--------
+
+An underscore "_" in pattern matches any single character, while a percent sign "%" matches any sequence of zero or more characters.
```

Comment:
Our implementation uses "?" and "*" in place of "_" and "%", respectively. I believe DIPS' implementation works so as well (@bjornna ?).

Reply:
Ok, I can change this, but I'm curious also about other implementations @bjornna @ppazos @serefarikan